### ISSUE TITLE
Configuration settings for html/css validation

### ISSUE THREAD

**Issue by justinribeiro** [Microsoft/typescript-lit-html-plugin#17]
@sjmiles noted this in https://github.com/mjbvz/vscode-lit-html/issues/42 

> I tried to solve this by disabling validation, i.e.
> 
> "html.validate.styles": false,
> "css.validate": false
> which had no effect.
> 
> Ideally ts-lit-html-plugin honors those configuration options.

With the update to `StyledTemplateLanguageService`, I'm wondering what your thoughts are on updating the `TsHtmlPluginConfiguration` interface to be enable `validate` and `lint` configuration options similar to styled? Or if there's a better configuration integration approach I'm not quite seeing.

**Comment by mjbvz** [comment]
Yes we should at least expose a way to disable validation. I propose a setting like:

```json
{
  "compilerOptions": {
    "plugins": [
      {
        "name": "typescript-lit-html-plugin",
        "css": { "validate": false }
      }
    ]
  }
}
```

The styled plugin has a number of settings that are also supported: https://github.com/Microsoft/typescript-styled-plugin#configuration
